portality/forms/application_forms.py

```diff
@@ -531,7 +531,25 @@ class FieldDefinitions:
         "help": {
             "long_help": ["The journal must use some form of licensing to be considered for indexing in DOAJ. ",
                           "If Creative Commons licensing is not used, then select <em>Publisher's own license</em> and enter "
-                          "more details below."],
+                          "more details below.",
```

Comment:
Looks good, worth checking rendering on the test server

Reply:
(This line adds another paragraph)

Let me know when it’s on test, but I’ve already viewed it locally.